client/components/exercises-mobile/exercise-accordion.jsx

```diff
@@ -13,8 +13,8 @@ export default class ExerciseAccordion extends React.Component {
     const muscleID = this.props.muscleID;
     fetch(`/api/exercises-by-muscle/${muscleID}`)
       .then(result => result.json())
-      .then(exercises => this.setState({ exercises }));
-    this.checkSavedIcons();
+      .then(exercises => this.setState({ exercises }))
+      .then(this.checkSavedIcons());
```

Comment:
`.then()` takes a function definition as its argument, but you are passing the return value of an (immediate) function _call_.